src/test/func/fixed_region/fixed_region.cc

```diff
@@ -32,8 +32,9 @@ int main()
 {
   MemoryProviderStateMixin<DefaultPal> mp;
 
-  size_t size = 1ULL << 28;
-  oe_base = mp.reserve<true>(&size, 0);
+  size_t large_class = 28 - SUPERSLAB_BITS;

```

Comment:
Not introduced in this change, but it would be nice to have 28 not be a magic number here.